src/splitEvery.js

```diff
@@ -0,0 +1,33 @@
+var _curry2 = require('./internal/_curry2');
+var slice = require('./slice');
+
+
+/**
+ * Splits a collection into slices of the specified length.
+ *
+ * Dispatches to its second argument's `slice` method if present. As a
+ * result, one may replace `[a]` with `String` in the type signature.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig Number -> [a] -> [[a]]
+ * @param {Number} n
+ * @param {Array} list
+ * @return {Array}
+ * @example
+ *
+ *      R.splitEvery(3, [1, 2, 3, 4, 5, 6, 7]); //=> [[1, 2, 3], [4, 5, 6], [7]]
+ *      R.splitEvery(3, 'foobarbaz'); //=> ['foo', 'bar', 'baz']
+ */
+module.exports = _curry2(function splitEvery(n, list) {
+  if (n <= 0) {
+    throw new Error('First argument to splitEvery must be a positive integer');
```

Comment:
@paldepind started a discussion about this in https://github.com/ramda/ramda/pull/1050#discussion_r31990842, objecting exactly to the behavior Highland has.  I don't have a strong sense of the right thing to do.  There's a big part of me that wants to return an empty list; it may not be the most logically correct thing, but it seems least harmful.  However there are some reasonable invariants that we could use to define lawful behavior on this function; those would likely be broken by returning an empty list.  Something like

    R.equals(ls, R.reduce(R.concat, [])(R.splitEvery(n, ls)));

Although I suppose we could just say, "for any list ls, for any positive integers n", that still feels odd.